lib/lens.ex

```diff
@@ -17,12 +17,14 @@ defmodule Lens do
   end
 
   @doc ~S"""
-  Returns a lens that focuses on the whole data.
+  Returns a lens that focuses on all of the data in the given scope.
```

Comment:
> Returns a lens that yields the entirety of the data currently under focus.

This sounds good to me. Does it address the problem you have? You could also split it into 2 sentences, one saying the general "all the data" thing and the other describing the behaviour in a chain of lenses.